geospatial/dted.ksy

```diff
@@ -0,0 +1,399 @@
+meta:
+  id: dted
+  title: DIGITAL TERRAIN ELEVATION DATA (DTED)
+  file-extension:
+    - dt0
+    - dt1
+    - dt2
+  endian: be
+  ks-version: 0.8
+doc-ref: |
+  https://dds.cr.usgs.gov/srtm/version2_1/Documentation/MIL-PDF-89020B.pdf
+doc: |
+  PERFORMANCE SPECIFICATION DIGITAL TERRAIN ELEVATION DATA (DTED). |
+  MIL-PRF-89020B (23 May 2000)
+seq:
+  - id: uhl
+    type: uhl_header
+  - id: dsi
+    type: dsi_header
+  - id: acc
+    type: acc_header
+  - id: records
+    type: data_record
+    repeat: expr
+    repeat-expr: uhl.number_of_longitude_lines.value
+types:
+  uhl_header:
+    seq:
+      - id: recognition_sentinel
+        contents: UHL
```

Comment:
Could you add `doc`s to everything?

Reply:
Docs will be added

Reply:
I don't know if this is right, but even single line comments I placed on a newline because it makes the web ids syntax highlighting easier to read. For example
```
      - id: longitude_interval_seconds
        type: ascii_int
        doc: |
          Longitude data interval in tenths of seconds.
```
instead of
```
      - id: longitude_interval_seconds
        type: ascii_int
        doc: Longitude data interval in tenths of seconds.
```

Reply:
IMHO: completely OK.